README.md

```diff
@@ -1,2 +1,48 @@
 # FIRE-WiFiCalling
 WiFi Calling App to be used with LibreServer
+
+## Environment configuration
+
+To configure basic dev environment
+
+First of all create an account in https://apps.ionic.io
+
+```
+sudo npm install -g ionic cordova
```

Comment:
This installed ionic 3.2.0 in my machine, we want v2, right? I did `$ sudo npm install -g ionic@2.2.3 cordova`